### PR TITLE
fmt: clang-format libfock

### PR DESCRIPTION
**Pull request by andysim**
## Description
clang-format libfock

## Questions
- [ ] If merging this PR will hinder your research b/c you have local edits, protest here or in #1144

## Status
- [x] Ready for review
- [x] Ready for merge

### CODE REVIEW

File: psi4/src/psi4/libfock/cubature.cc [hunk 1]
```diff
@@ -53,18 +52,15 @@ using namespace psi;
...
-        1.000,
-        1.001,                                                                                                                 1.012,
-        0.825, 1.408,                                                                       1.485, 1.452, 1.397, 1.342, 1.287, 1.243,
-        1.144, 1.364,                                                                       1.639, 1.716, 1.705, 1.683, 1.639, 1.595,
-        1.485, 1.474, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.650, 1.727, 1.760, 1.771, 1.749, 1.727,
-        1.628, 1.606, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.672, 1.804, 1.881, 1.892, 1.892, 1.881,
+        1.000, 1.001, 1.012, 0.825, 1.408, 1.485, 1.452, 1.397, 1.342, 1.287, 1.243, 1.144, 1.364, 1.639,
+        1.716, 1.705, 1.683, 1.639, 1.595, 1.485, 1.474, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562, 1.562,
+        1.562, 1.562, 1.562, 1.650, 1.727, 1.760, 1.771, 1.749, 1.727, 1.628, 1.606, 1.639, 1.639, 1.639,
+        1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.639, 1.672, 1.804, 1.881, 1.892, 1.892, 1.881,
```
Comment (loriab) [inline psi4/src/psi4/libfock/cubature.cc]: should unformat this block

File: psi4/src/psi4/libfock/cubature.cc [hunk 2]
```diff
@@ -2627,56 +2560,44 @@ void StandardGridMgr::Initialize_SG1()
...
+    static const PruneGroup row3spec[] = {{6, 12}, {38, 7}, {86, 5}, {194, 7}, {86, 19}, {0, 0}};  // Na to Ar
+    static const PruneGroup row4spec[] = {{38, 12}, {194, 38}, {0, 0}};  // Everything after Argon
 
     // rows[Z] tells you on which row of the periodic table you can find element Z.
     static const short rows[] = {
-        0,
-        1,                      1,
-        2, 2,    2, 2, 2, 2, 2, 2,
-        3, 3,    3, 3, 3, 3, 3, 3,
+        0, 1, 1, 2, 2, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 3, 3,
```
Comment (loriab) [inline psi4/src/psi4/libfock/cubature.cc]: unformat

File: psi4/src/psi4/libfock/cubature.cc (hunk 1 above)
Comment (andysim) [reply psi4/src/psi4/libfock/cubature.cc]: Good catch; I just blindly ran c-f.  Will have the changes in before PR merge deadline.

File: psi4/src/psi4/libfock/cubature.cc (hunk 2 above)
Comment (andysim) [reply psi4/src/psi4/libfock/cubature.cc]: 10-4

**Comment by loriab** [comment]
superseded by #1397

**Pull request closed without merging**